### PR TITLE
Moved now config to package.json

### PR DESCRIPTION
**Pull request by spiray**
@sbibow Is there a way for us to test the deploy script locally to make sure that this change didn't break anything?

### CODE REVIEW

**Comment by ewenjo** [comment]
You can use the `now dev` command to run the app locally using now.sh builders

**Comment by ewenjo** [comment]
I think `now.json` only be inside `package.json` if we are seving static files.

**Comment by spiray** [comment]
@ewenjo That makes sense. I switched it back to the now.json file and kept the other changes.

File: src/app.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ const app = express();
 app.use(express.json());
 
 // App routes
-app.get('/', (req, res) => res.json({ msg: 'Check out /contributors, /faq, /history and /video' }));
+app.get('/', (req, res) => res.json({ message: 'Check out /contributors, /faq, /history and /video' }));
```
Comment (ewenjo) [inline src/app.js]: Should we add `/docs`?

**Review by ewenjo — APPROVED** [review]

**Pull request merged**